src/entities/OrderTax.java

```diff
@@ -0,0 +1,43 @@
+package entities;
+
+public class OrderTax extends BaseEntity<Long>{
```

Comment:
I think you can create a base tax history class that you could use for OrderDetailTax and OrderTax

Reply:
with attributes like name, percentage, and original tax that ir refers too? BaseTaxEntity should also extend from BaseEntity

Reply:
Yes